browser/ui/views/split_view/split_view_location_bar_browsertest.cc

```diff
@@ -27,56 +26,29 @@
 #include "content/public/test/browser_test_utils.h"
 #include "net/dns/mock_host_resolver.h"
 
-// Run with two modes(brave split view or chromium side by side).
-class SplitViewLocationBarBrowserTest
-    : public InProcessBrowserTest,
-      public testing::WithParamInterface<bool> {
+class SplitViewLocationBarBrowserTest : public InProcessBrowserTest {
```

Comment:
changed this test only with brave split view as this location bar is not used by SideBySide anymore.